==== triton/include/triton/Dialect/NVGPU/IR/NVGPUOps.td#3 - /google/src/cloud/shyshkov/mlir_4983432f17eb4b445e161c5f8278c6ea4d5d1241_1699531174/triton/include/triton/Dialect/NVGPU/IR/NVGPUOps.td ====
# action=edit type=text
--- triton/include/triton/Dialect/NVGPU/IR/NVGPUOps.td	2023-11-09 03:52:05.000000000 -0800
+++ triton/include/triton/Dialect/NVGPU/IR/NVGPUOps.td	2023-11-09 04:00:14.000000000 -0800
@@ -29,8 +29,8 @@
 include "mlir/Dialect/LLVMIR/LLVMOpBase.td"
 include "mlir/Interfaces/InferTypeOpInterface.td" // SameOperandsAndResultType
 
-def LLVM_PointerGlobal : LLVM_OpaquePointerInAddressSpace<1>;
-def LLVM_PointerShared : LLVM_OpaquePointerInAddressSpace<3>;
+def LLVM_PointerGlobal : LLVM_PointerInAddressSpace<1>;
+def LLVM_PointerShared : LLVM_PointerInAddressSpace<3>;
 
 class NVGPU_Op<string mnemonic, list<Trait> traits = []> :
     LLVM_OpBase<NVGPU_Dialect, mnemonic, traits>;
